src/ontology/edits/oeo-physical.omn

```diff
@@ -9262,6 +9262,121 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1359",
+        <http://purl.obolibrary.org/obo/RO_0002427> some OEO_00310027
+    
+    
+Class: OEO_00310027
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A gas turbine process is an energy transformation that converts chemical energy into rotational energy using a continous internal combustion process. A gas turbine is participating in a gas turbine process.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "gas turbine process"
+    
+    SubClassOf: 
+        OEO_00020003,
+        OEO_00000532 some OEO_00000099,
+        OEO_00010234 some OEO_00000007,
+        OEO_00010235 some OEO_00240000,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00140038,
+        <http://purl.obolibrary.org/obo/RO_0000057> some OEO_00000185
+    
+    
+Class: OEO_00310028
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A steam power unit is a power generating unit using steam.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "steam power unit"
+    
+    SubClassOf: 
+        OEO_00000334,
+        OEO_00000503 some OEO_00110001,
+        OEO_00010234 some OEO_00000007,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000396
+    
+    
+Class: OEO_00310029
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A combined cycle power plant (CCGT) is a gas power plant that has gas fired power units, steam power units, gas power units and heat recovery steam units as parts.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "CCGT",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "combined cycle power plant"
+    
+    SubClassOf: 
+        OEO_00000184,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00310028,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00310030,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00310031
+    
+    
+Class: OEO_00310030
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A gas power unit is a power generating unit using gas.",
```

Comment:
How is that different from `gas fired power unit`?  Why are both required?

Reply:
`gas power unit` has as part a `gas turbine` which converterts chemical energy to rational energy.
Whereas `gas fired power unit` has as part a `electro motive generator` which converts kinetic energy to electric energy.
But I now think that a CCGT also has `gas fired power unit` as parts, so I will extend the definition there.

Reply:
I followed this diagramm: 
![image](https://user-images.githubusercontent.com/30029360/205618836-c76f6e82-0706-45b1-9185-9eafc5efe473.png)
![image](https://user-images.githubusercontent.com/30029360/205618933-4b241cec-dc13-463d-85c6-01df8d8ffda1.png)

Reply:
`steam power unit` is still a useful concept.